### ISSUE TITLE
IDEA

### ISSUE THREAD

**Issue by MrRevillod** [vinceliuice/WhiteSur-gtk-theme#744]
Can you make a centering to the url bar firefox theme? I mean like safari

**Comment by vinceliuice** [comment]
https://github.com/vinceliuice/WhiteSur-gtk-theme/tree/master/src/other/firefox

**Comment by MrRevillod** [comment]
yes I have that theme, but I talk about the text in the url bar and its behavior, it's just a suggestion, thank you very much for your work

**Comment by vinceliuice** [comment]
![截图 2022-11-28 16-27-31](https://user-images.githubusercontent.com/7604295/204229764-1c21a33c-2e67-47a8-9fb3-2627310a52fd.png)
